src/pages/SendNft.vue

```diff
@@ -331,7 +445,7 @@ watch(
         gasFee: gasFeeInEth,
         estimatedGas,
         nftContractAddress: props.address,
-        tokenId: props.tokenId,
+        tokenId: props.tokenId || props.identifier as string,
```

Comment:
Is identifier an id or collectionName?

ref: https://github.com/arcana-network/wallet-ui/pull/615/files#r1507869776

Reply:
identifier in mvx is `collection-nonce` but `collection` and `nonce` should be passed separately.